config/initializers/good_job.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
I'm wondering if we should define some configs here, maybe creating a "high_priority" queue? No need to change now, just thinking out loud

Reply:
ahh right, I'd go with 3 queues by default: high, default, low

Reply:
I was checking, and by default good job will process all jobs from all queues. Do you want to add 

`config.good_job.queues = "high;default;low"`

to limit the queues that we are going to process? We also need to take care of ActiveStorage (and others like mailers?) Rails queues I guess?

Reply:
What about
```ruby
config.good_job.queues = "high;default;low;*"
```

I think this will cover all the scenarios, but also assign more "priority" to the queues we define.

Reply:
what I mean is that we don't need to configure queues, it's just for the workers to decide which queues they process

Reply:
hmm ok but I guess we don't want to process low before others(e.g. mailers)

Reply:
Agree it is not needed, just though it would be nice to have a queue that will take higher priority than a mailer for instance if we want to. But I'm ok with no providing defaults for this anyway, it can be defined if needed

Reply:
can you check now? I put `  config.good_job.queues = "high;*"` by default, and also enabled this option by default https://github.com/bensheldon/good_job/pull/1311 to not depend on transactions